src/Sql/Dirt/dbo/Tables/RiskInsightReport.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE [dbo].[RiskInsightReport] (
+    [Id]                       UNIQUEIDENTIFIER NOT NULL,
+    [OrganizationId]           UNIQUEIDENTIFIER NOT NULL,
+    [Date]                     DATETIME2 (7)    NOT NULL,
+    [ReportData]               NVARCHAR(MAX)    NOT NULL,
+    [CreationDate]             DATETIME2 (7)    NOT NULL,
+    [RevisionDate]             DATETIME2 (7)    NOT NULL,
+    CONSTRAINT [PK_RiskInsightReport] PRIMARY KEY CLUSTERED ([Id] ASC),
+    CONSTRAINT [FK_RiskInsightReport_Organization] FOREIGN KEY ([OrganizationId]) REFERENCES [dbo].[Organization] ([Id])
```

Comment:
```suggestion
    CONSTRAINT [FK_RiskInsightReport_Organization] FOREIGN KEY ([OrganizationId]) REFERENCES [dbo].[Organization] ([Id]) ON DELETE CASCADE 
```
This is needed because when an organization is deleted, it must delete records in this table as well.